### PR TITLE
Am/feat/expand no verify

### PR DESCRIPTION
**Pull request by IceTDrinker**

### CODE REVIEW

File: tfhe/src/high_level_api/compact_list.rs [hunk 1]
```diff
@@ -292,6 +292,60 @@ mod zk {
                 Some(_) => Err(crate::Error::new("Expected a CPU server key".to_string())),
             })
         }
+
+        #[doc(hidden)]
+        /// This function allows to expand a ciphertext without verifying the associated proof.
+        ///
+        /// If you are here you were probably looking for it: use at your own risks.
+        pub fn expand_without_verification(&self) -> crate::Result<CompactCiphertextListExpander> {
```
[the remark this answers is not in the document]
Comment (IceTDrinker) [reply tfhe/src/high_level_api/compact_list.rs:300]: good point, I was wondering about it while I was writing this down, now that I think about it again it should be straightforward indeed

File: tfhe/src/integer/ciphertext/compact_list.rs [hunk 2]
```diff
@@ -636,6 +636,62 @@ impl ProvenCompactCiphertextList {
...
+                        }
+                    }
+
+                    extract_message_and_carries(expanded_blocks, sks)
+                }
+                IntegerCompactCiphertextListUnpackingMode::NoUnpacking => unreachable!(),
+            }
+        } else {
+            expanded_blocks
+        };
```
[the remark this answers is not in the document]
Comment (IceTDrinker) [reply tfhe/src/integer/ciphertext/compact_list.rs:687]: I will take another look, but you are most likely correct

[the remark this answers is not in the document]
Comment (IceTDrinker) [reply tfhe/src/integer/ciphertext/compact_list.rs:687]: as discussed the extracted code is too small in the end to be worth it, at least for now

File: tfhe/src/high_level_api/compact_list.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (IceTDrinker) [reply tfhe/src/high_level_api/compact_list.rs:300]: done in shortint as discussed, harder to have it in the HL due to the different APIs (with different arguments passed as inputs)

**Review by nsarlin-zama — APPROVED** [review]

**Comment by IceTDrinker** [comment]
fixed a js/wasm crash due to a bad change in the worker.js file, will relaunch

**Comment by IceTDrinker** [comment]
https://github.com/zama-ai/tfhe-rs/actions/runs/10848966475

**Pull request merged**